### PR TITLE
[4.0] com_users tfa hover

### PR DESCRIPTION
**Pull request by brianteeman**
PR for #30000

### Before
before

### After
tfa

### CODE REVIEW

File: administrator/components/com_users/tmpl/users/default.php [hunk 1]
```diff
@@ -156,14 +156,16 @@
...
 								<?php if ($tfa) : ?>
-								<td class="text-center d-none d-md-table-cell tbody-icon">
+								<td class="text-center d-none d-md-table-cell">
+									<span class="tbody-icon">
 									<?php if (!empty($item->otpKey)) : ?>
 										<span class="fas fa-check" aria-hidden="true"></span>
 										<span class="sr-only"><?php echo Text::_('COM_USERS_TFA_ACTIVE'); ?></span>
 									<?php else : ?>
-										<span class="fas fa-times" aria-hidden="true"></span>
+										<span class="tbody-icon fas fa-times" aria-hidden="true"></span>
```
Comment (Quy) [inline administrator/components/com_users/tmpl/users/default.php]:
```suggestion
										<span class="fas fa-times" aria-hidden="true"></span>
```

Comment (brianteeman) [reply administrator/components/com_users/tmpl/users/default.php]: Why?

**Comment by toivo** [comment]
I have tested this item :white_check_mark: successfully on 2f377f04b0b95f3465220ce1a7013567ed7bbe63<br /><br />Tested successfully in Beta3-dev of 6 July.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30003">issues.joomla.org/tracker/joomla-cms/30003</a>.</sub>

**Review by brianteeman — COMMENTED** [review]
Sorry. I thought you were adding it. Quite correct to remove it. I committed the wrong one

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 16cfbbd959821bcff7965bacea9abc074232e387<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30003">issues.joomla.org/tracker/joomla-cms/30003</a>.</sub>

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 16cfbbd959821bcff7965bacea9abc074232e387<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30003">issues.joomla.org/tracker/joomla-cms/30003</a>.</sub>

**Comment by jwaisner** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30003">issues.joomla.org/tracker/joomla-cms/30003</a>.</sub>

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**